### PR TITLE
COM-3634 Added order cancel failed email template

### PR DESCRIPTION
**Pull request by SharmaBhushan**

### CODE REVIEW

File: templates/email/order-cancel-failed.hypr [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  </div>  
+  <div class="header-text">
+    <div class="store-name">{{ siteContext.generalSettings.websiteName }}</div>
+  </div>
+</div>
+<br />
+<div>
+    <br /><br />
+    {{ labels.orderCancelFailed|string_format(model.orderNumber)|safe }} 
+     <a href="//{{ adminDomainName }}/Admin/s-{{model.siteId}}/orders/edit/{{model.id}}">http://{{ adminDomainName }}/Admin/s-{{model.siteId}}/orders/edit/{{model.id}}</a>.
```
Comment (hrishid) [inline templates/email/order-cancel-failed.hypr]: Add https to the url as discussed in dev call.

File: labels/en-US.json [hunk 2]
```diff
@@ -484,6 +484,7 @@
   "order": "Order",
   "orderCancelBlob1": "The following item(s) from order #{0}, external order #{1} have been cancelled.",
```
Comment (hrishid) [inline labels/en-US.json:485]: Not part of your story but we are missing translation here..! orderCancelBlob1 and orderCancelBlob2

Comment (SharmaBhushan) [reply labels/en-US.json:485]: Done.

File: templates/email/order-cancel-failed.hypr (hunk 1 above)
Comment (SharmaBhushan) [reply templates/email/order-cancel-failed.hypr]: Done

File: templates/email/order-cancel-failed.hypr [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+<!--- Header --->
+<div class="mz-b2b-email-header">
```
Comment (ghost) [inline templates/email/order-cancel-failed.hypr]: Why is this using the B2B email header? This is not a B2B email. Use the general header for normal emails.

File: templates/email/order-cancel-failed.hypr [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  </div>  
+  <div class="header-text">
+    <div class="store-name">{{ siteContext.generalSettings.websiteName }}</div>
+  </div>
+</div>
+<br />
+<div>
+    <br /><br />
+    {{ labels.orderCancelFailed|string_format(model.orderNumber)|safe }} 
+     <a href="//{{ adminDomainName }}/Admin/s-{{model.siteId}}/orders/edit/{{model.id}}">https://{{ adminDomainName }}/Admin/s-{{model.siteId}}/orders/edit/{{model.id}}</a>.
```
Comment (ghost) [inline templates/email/order-cancel-failed.hypr]: Need to add `https` to the `a href` not just to the text.

File: templates/email/order-cancel-failed.hypr [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  <div class="header-logo">
+    <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+  </div>  
+  <div class="header-text">
+    <div class="store-name">{{ siteContext.generalSettings.websiteName }}</div>
+  </div>
+</div>
+<br />
+<div>
+    <br /><br />
```
Comment (ghost) [inline templates/email/order-cancel-failed.hypr]: Are 2 breaks really necessary? I don't see this being doubled up in other templates I just checked.

**Review by ghost — CHANGES_REQUESTED** [review]
Overall looks fine but need to fix a few things. See my comments.

File: templates/email/order-cancel-failed.hypr (hunk 3 above)
Comment (SharmaBhushan) [reply templates/email/order-cancel-failed.hypr]: I checked non b2b templates there we are not using header so I removed header html part.

File: templates/email/order-cancel-failed.hypr (hunk 5 above)
Comment (SharmaBhushan) [reply templates/email/order-cancel-failed.hypr]: Removed 'br' tag. Only for add some bottom space was using 'br' tag.

File: templates/email/order-cancel-failed.hypr (hunk 4 above)
Comment (SharmaBhushan) [reply templates/email/order-cancel-failed.hypr]: Done.

**Review by ghost — APPROVED** [review]

**Pull request merged**